storybook/src/stories/Form/FileManagement/AreaUploadManager.stories.tsx

```diff
@@ -26,7 +26,8 @@ export const _AreaUploadManager = () => {
   const addMoreFilesText = text('Add More Files Text', 'Add More files')
   const fileIcons = object('File Icons', defaultIcons)
   const showValue = action('Input Callback');
-  const allowedFileTypes = object('allowedFileTypes', fileTypes);
+  const allowedFileTypes = object('Allowed File Types', fileTypes);
```

Comment:
The existing component is not working with allowedFileTypes. Is it fixed in this PR?

Reply:
It is working